styles/Canonical/003-Ubuntu-names-versions.yml

```diff
@@ -0,0 +1,34 @@
+extends: existence
+message: |
+  Please use the Ubuntu product name correctly. E.g. Ubuntu 22.04 LTS, Ubuntu
+  22.10, Ubuntu 23.04 (Lunar Lobster), Ubuntu 23.10 (Mantic), Ubuntu 24.04 LTS
+  (Noble Numbat).
+link: https://docs.ubuntu.com/styleguide/en/#ubuntu
+nonword: true
+ignorecase: true
+scope: 
+  - sentence
+  - heading
+  - table 
+level: suggestion
+tokens:
+  # LTS releases should be followed by "LTS" suffix.
+  - 'Ubuntu (24|22|20|18)\.04(?! LTS)'
+  # Non-LTS releases should not be followed by "LTS" suffix.
+  - 'Ubuntu (23\.10|23\.04|22\.10|21\.10|21\.04|20\.10|19\.10|19\.04|18\.10|17\.10|17\.04) LTS'
+  # All releases should match the code name.
+  - 'Ubuntu 24\.04 LTS \((?!(Noble|Noble Numbat)\))'
+  - 'Ubuntu 23\.10 \((?!(Mantic|Mantic Minotaur)\))'
+  - 'Ubuntu 23\.04 \((?!(Lunar|Lunar Lobster)\))'
+  - 'Ubuntu 22\.10 \((?!(Kinetic|Kinetic Kudu)\))'
+  - 'Ubuntu 22\.04 LTS \((?!(Jammy|Jammy Jellyfish)\))'
+  - 'Ubuntu 21\.10 \((?!(Impish|Impish Indri)\))'
+  - 'Ubuntu 21\.04 \((?!(Hirsute|Hirsute Hippo)\))'
+  - 'Ubuntu 20\.10 \((?!(Groovy|Groovy Gorilla)\))'
+  - 'Ubuntu 20\.04 LTS \((?!(Focal|Focal Fossa)\))'
+  - 'Ubuntu 19\.10 \((?!(Eoan|Eoan Ermine)\))'
+  - 'Ubuntu 19\.04 \((?!(Disco|Disco Dingo)\))'
+  - 'Ubuntu 18\.10 \((?!(Cosmic|Cosmic Cuttlefish)\))'
+  - 'Ubuntu 18\.04 LTS \((?!(Bionic|Bionic Beaver)\))'  
+
```

Comment:
It would be nice to have a complete list but maybe we can make that an improvement as a straightforward ODA task

Reply:
I can actually generate the complete list easily but it's more of a maintenance thing, e.g. it would be nice to automate this by grabbing the information from ubuntu.com.

I'll push an update in another PR.

Thanks!